client/sender.go

```diff
@@ -31,15 +33,17 @@ import (
 type KVSender interface {
 	// Send invokes the Call.Method with Call.Args and sets the result
 	// in Call.Reply.
-	Send(Call)
+	Send(context.Context, Call)
 }
 
 // KVSenderFunc is an adapter to allow the use of ordinary functions
 // as KVSenders.
 type KVSenderFunc func(Call)
 
+var _ KVSender = KVSenderFunc(func(Call) {})
+
 // Send calls f(c).
-func (f KVSenderFunc) Send(c Call) {
+func (f KVSenderFunc) Send(_ context.Context, c Call) {
```

Comment:
Add Context as an argument to KVSenderFunc as well (there's only one place in the tests that needs to be updated. We don't really need the context there, but KVSenderFunc is intended to have the same signature as KVSender.Send)